wagtail/admin/views/reports/audit_logging.py

```diff
@@ -52,62 +50,35 @@ def get_actions_for_filter(user):
     ]
 
 
-class SiteHistoryReportFilterSet(WagtailFilterSet):
-    action = django_filters.ChoiceFilter(
-        label=_("Action"),
-        # choices are set dynamically in __init__()
-    )
-    hide_commenting_actions = django_filters.BooleanFilter(
-        label=_("Hide commenting actions"),
-        method="filter_hide_commenting_actions",
-        widget=forms.CheckboxInput,
-    )
-    timestamp = django_filters.DateFromToRangeFilter(
-        label=_("Date"), widget=DateRangePickerWidget
-    )
+class SiteHistoryReportFilterSet(PageHistoryFilterSet):
     label = django_filters.CharFilter(label=_("Name"), lookup_expr="icontains")
-    user = django_filters.ModelChoiceFilter(
-        label=_("User"),
-        field_name="user",
-        queryset=lambda request: get_users_for_filter(request.user),
-    )
     object_type = ContentTypeFilter(
         label=_("Type"),
         method="filter_object_type",
         queryset=lambda request: get_content_types_for_filter(request.user),
     )
 
-    def filter_hide_commenting_actions(self, queryset, name, value):
-        if value:
-            queryset = queryset.exclude(action__startswith="wagtail.comments")
-        return queryset
-
     def filter_object_type(self, queryset, name, value):
         return queryset.filter_on_content_type(value)
 
+    def get_action_choices(self):
+        return get_actions_for_filter(self.request.user)
+
+    def get_user_choices(self):
+        return get_users_for_filter(self.request.user)
+
     class Meta:
         model = PageLogEntry
-        fields = [
-            "object_type",
-            "label",
-            "action",
-            "user",
-            "timestamp",
-            "hide_commenting_actions",
```

Comment:
All of these filters are defined on the filterset class, not automatically derived the model, so specifying them here is a bit misleading.